packages/create-novu-app/index.ts

```diff
@@ -42,11 +42,11 @@ const program = new Commander.Command(packageJson.name)
     projectPath = name;
   })
   .option(
-    '-k, --api-key [apiKey]',
+    '-k, --secret-key [secretKey]',
     `
 
-  Your Novu Development environment apiKey. Note that your novu app won't
-  work outside of local mode without an apiKey.
+  Your Novu development environment secret key. Note that your novu app won't
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏ this can also be the prod secret key, maybe remove the name of the env

Reply:
We shouldn't promote the use of prod key at this point.